database/copy.js

```diff
@@ -0,0 +1,39 @@
+//how to run schema: \i /Users/kianna/Desktop/review-module/schema.sql
+const pg = require('pg');
+const connectionString = 'postgres://localhost:5432/project_nomad_reviews';
+
+/*************************** CONNECT TO POSTGRES *****************************/
+const client = new pg.Client(connectionString);
+
+client.connect((err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('you have succeeded in seeding');
+  }
+});
+
+const query = `COPY listings FROM '/Users/Kianna/Desktop/review-module/database/listing_sample.csv' DELIMITER ',' CSV HEADER`;
+client.query(query, (err) => {
+  if (err) {
+    throw err;
+  } else {
+    console.log('listing');
```

Comment:
You could be more descriptive in your success log. Imagine if someone ran your code and 'listing' popped up in console